k8s/namespaces/neuvector/patches/stg/neuvector.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
   namespace: neuvector
 spec:
   values:
+    neuvector:
```

Comment:
didn't you enable this by default? https://github.com/hmcts/chart-neuvector/blob/master/neuvector-azure-keyvault/values.yaml#L35-L36

Reply:
we do. Just trying to keep it in line with CFT

Reply:
Neuvector pods are failing to start. Just want to keep it consistent.

Reply:
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/neuvector/patches/prod/neuvector.yaml

Reply:
CFT has it because they have a cluster not on containerd yet, thomas is upgrading it atm.

so you shouldn't make that change for the sake of it